tests/mmptest/src/MMPTest.cs

```diff
@@ -1,3 +1,5 @@
+// #define ENABLE_STATIC_REGISTRAR_TESTS https://bugzilla.xamarin.com/show_bug.cgi?id=58629
```

Comment:
The only "problem" is with tests like UnifiedWithoutLinking_ShouldHaveManyFrameworkClangLines and UnifiedWithLinking_ShouldHaveFewFrameworkClangLines that have static registrar parts. 

We'd have to split them, though this is not hard.